backend/typescript/prisma/schema.prisma

```diff
@@ -98,7 +98,7 @@ model staff {
   profilePictureLink String?        @map("profile_picture_link")
   tasks_assigned     task[]
   warnings_assigned  warning[]
-  notifications      notification[]
+  // notifications      notification[]
```

Comment:
Maybe it might make sense to add a TODO comment in this file to uncomment these lines when the staff apis are created, as it might be hard to understand why these lines are commented otherwise